src/content/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/monitor-services/monitor-services-running-kubernetes.mdx

```diff
@@ -276,7 +242,12 @@ This automatic discovery works the same as the [container auto-discovery](/docs/
 
 ### Add a service YAML to the Kubernetes integration config [#configmap]
 
-In order for the snippet above to be recognized by the integration, it must be placed inside the designated ConfigMap. If you are using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3), a ConfigMap should already be generated, with a name ending in `-integrations-cfg`. Locate the config map and add the modified snippet to it, so it ends up looking like this:
+In order for the snippet above to be recognized by the integration, it must be placed inside the designated ConfigMap. If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), a ConfigMap should already be generated, with a name ending in `-integrations-cfg`. 
```

Comment:
```suggestion
To be recognized by the integration, the code snippet must be placed within a designated `ConfigMap`. If you're using our [Kubernetes integration v3](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/k8s-version2/changes-since-v3), a `ConfigMap` with a name ending in `-integrations-cfg` should already exist.
```

Reply:
Check other references of ConfigMap on this page and make sure they're wrapped in code format

Reply:
I've rewritten the first sentence to avoid the passive form:
`The integration will recognize the above snippet, if you place it inside a designated `ConfigMap``
I checked the other references of ConfigMap and now they are all in code format! Thank you!!